standalone/src/data_conversion_layer/start_request_serialization.cpp

```diff
@@ -95,9 +98,19 @@ RawData data_conversion_layer::start_request::serialize(const data_conversion_la
   raw_processing::write(os, speed_encoder_enabled);
   raw_processing::write(os, diagnostics_enabled);
 
-  raw_processing::write(os, msg.master_.getScanRange().getStart().value());
-  raw_processing::write(os, msg.master_.getScanRange().getEnd().value());
-  raw_processing::write(os, msg.master_.getResolution().value());
+  const auto start = msg.master_.getScanRange().getStart().value();
+  const auto end = msg.master_.getScanRange().getEnd().value();
+  const auto resolution = msg.master_.getResolution().value();
+
+  raw_processing::write(os, start);
+  raw_processing::write(os, end);
+  raw_processing::write(os, resolution);
+
+  PSENSCAN_INFO("StartRequestSerialization",
+                "Serializing start request with angle_start={} angle_end={} resolution={} tenths of degree.",
```

Comment:
Either degrade to `_DEBUG` or use a message, that the user easily understands. E.g. "Starting scanner with range [22.4°, 44.8°] @0.1° resolution". Especially hide `TenthOfDegree` and "Serializing" from the end user.

Reply:
I degraded to DEBUG, because the information we need indeed are the low level parameters sent in the start request to the scanner.